src/cmdc/datasets/uscensus/schema.sql

```diff
@@ -0,0 +1,71 @@
+DROP SCHEMA IF EXISTS uscensus CASCADE;
+
+CREATE SCHEMA uscensus;
+
+/* Create and define metadata tables */
+CREATE TABLE uscensus.acs_variables (
+    "id" SERIAL UNIQUE,
+    "year" INTEGER,
+    "product" CHAR(4),
+    "census_id" VARCHAR(20),
+    "label" VARCHAR(350),
+    PRIMARY KEY ("year", "product", "census_id")
+);
+COMMENT ON TABLE uscensus.acs_variables IS E'
+Table Description:
+
+This table contains information on the variables that are listed for one of the ACS products (1 and 5 year estimates):
+
+For more information on these variables, please refer to the US Census page about the [ACS 1 year estimates](https://www.census.gov/data/developers/data-sets/acs-1year.html) and the [ACS 5 year estimates](https://www.census.gov/data/developers/data-sets/acs-5year.html)
+
+Words of caution:
```

Comment:
Any words of caution??

Reply:
Not yet -- I was trying to create a common structure that we could use for documenting all of the tables and it seemed like `Words of caution` is a useful thing to have in the documentation to make sure that people understand the caveats of certain data sets... I haven't filled it in for any of these tables yet though